server/src/main/java/io/druid/server/lookup/cache/LookupCoordinatorManager.java

```diff
@@ -87,13 +85,13 @@
  */
 public class LookupCoordinatorManager
 {
-  public static final String LOOKUP_CONFIG_KEY = "lookups";
-  // Doesn't have to be the same, but it makes things easy to look at
-  public static final String LOOKUP_LISTEN_ANNOUNCE_KEY = LOOKUP_CONFIG_KEY;
+  //key used in druid-0.9.x with config manager
+  public static final String OLD_LOOKUP_CONFIG_KEY = "lookups";
+
+  public static final String LOOKUP_CONFIG_KEY = "lookupsConfig";
+  public static final String LOOKUP_LISTEN_ANNOUNCE_KEY = "lookups";
```

Comment:
listen announce key is completely independent of the config key, i think its fine for it to keep the separation.